functests/2_performance_update/updating_profile.go

```diff
@@ -55,6 +56,67 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	Context("Verify GloballyDisableIrqLoadBalancing Spec field", func() {
+		It("[test_id:36150] Verify that IRQ load balancing is enabled/disabled correctly", func() {
+			irqLoadBalancingDisabled := profile.Spec.GloballyDisableIrqLoadBalancing != nil && *profile.Spec.GloballyDisableIrqLoadBalancing
+
+			profileIsolatedCPUs := string(*profile.Spec.CPU.Isolated)
+			isolatedCPUsMask, err := components.CPUListToHexMask(profileIsolatedCPUs)
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("failed to list in Hex %s: %v", profileIsolatedCPUs, err))
+			isolatedCPUs, err := strconv.ParseUint(isolatedCPUsMask, 16, 64)
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("failed to parse %s: %v", isolatedCPUsMask, err))
+
+			var expectedBannedCPUs uint64
+			if irqLoadBalancingDisabled {
+				expectedBannedCPUs = isolatedCPUs
+			}
+
+			for _, node := range workerRTNodes {
+				bannedCPUs, err := nodes.BannedCPUs(node)
+				Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("failed to extract the banned CPUs from node %s: %v", node.Name, err))
+				Expect(bannedCPUs).To(Equal(expectedBannedCPUs))
+			}
+
+			By("Modifying profile")
+			initialProfile = profile.DeepCopy()
+
+			irqLoadBalancingDisabled = !irqLoadBalancingDisabled
+			if irqLoadBalancingDisabled {
+				expectedBannedCPUs = isolatedCPUs
+			} else {
+				expectedBannedCPUs = 0
+			}
+			profile.Spec.GloballyDisableIrqLoadBalancing = &irqLoadBalancingDisabled
+
+			By("Updating the performance profile")
+			Expect(testclient.Client.Update(context.TODO(), profile)).ToNot(HaveOccurred())
+			defer func() { // return initial configuration
+				spec, err := json.Marshal(initialProfile.Spec)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(testclient.Client.Patch(context.TODO(), profile,
+					client.RawPatch(
+						types.JSONPatchType,
+						[]byte(fmt.Sprintf(`[{ "op": "replace", "path": "/spec", "value": %s }]`, spec)),
+					),
+				)).ToNot(HaveOccurred())
```

Comment:
do we need to wait here?

Reply:
We updated the profile. Now the tuned profile will be updated and the node will be updated. It happens async, right?

Reply:
Following the other snippet I would say no. The update is very "fast". If we do wait, we should wait on both, right?

Reply:
ok, just checking

Reply:
Once we re-write, we can add the wait as a good practice though.